src/huntsman/pocs/dome/musca.py

```diff
@@ -113,7 +113,7 @@ def door_closed(self):
     @property
     def is_safe_to_open(self):
         v = self.status[Protocol.BATTERY]
-        if v < self.MIN_OPERATING_VOLTAGE:
+        if float(v) < self.MIN_OPERATING_VOLTAGE:
```

Comment:
This is the wrong place for this change. The function that gets the status for the dome should be converting number string to floats before it returns the status dict.

Reply:
In this function, should convert `status[Protocol.BATTERY]` and probably `status[Protocol.SOLAR_ARRAY]` to floats:

https://github.com/AstroHuntsman/huntsman-pocs/blob/4949428c09eb27f08c19f516a35203e5dc7db711/src/huntsman/pocs/dome/musca.py#L122